### DISCUSSION TITLE
Comparing pole figures/odfs

### DISCUSSION THREAD

**Discussion by KOuadie** [mtex-toolbox/mtex#607] [Ask Anything]
Hello everyone,
I am a beginner level user. I have EBSD and XRD data, and my goal is to compare between the odfs/pole figures obtained with XRD and EBSD for the same material [dual-phase (Ferrite and Austenite) steel].
I have 6 EBSD datasets that I combined. 
How can I compare between the odfs and pole figures ? is there a code or an option in MTEX that would help me compare between the pole figures/odfs ? I was thinking maybe like differential imaging ?
I am using MatLAB R2020a and MTEX5.5.beta.3.
Thank you in advance for your help.
Best regards,
K. Ouadie

**Comment by kilir** [comment]
Hi,
do I understand you want to compare an odf against another odf and a pf against another pf, right?
You could subtract from or divide by each other.

Cheers,
Rüdiger
